WebDevStudios/constant-contact-forms
# CC-31: Add support for multiple reCAPTCHA checks on the same page

Ticket: [CC-31](https://webdevstudios.atlassian.net/browse/CC-31)

## Description ##

Adds ability to validate multiple reCAPTCHA fields on the same page.

## Testing ##

- Enable reCAPTCHA v2 in CC settings (Contact Form > Settings > Spam Control).
- Add at least two CC forms to one page.
- Validate one form with reCAPTCHA and confirm:
    - Validated form allows submission ("Sign up" button is enabled).
    - Other (non-validated) forms do not allow submission ("Sign up" buttons are disabled).
